packages/eslint-plugin-next/lib/index.js

```diff
@@ -28,6 +29,7 @@ module.exports = {
         '@next/next/google-font-display': 'warn',
         '@next/next/google-font-preconnect': 'warn',
         '@next/next/next-script-for-ga': 'warn',
+        '@next/next/no-assign-module-variable': 'warn',
```

Comment:
```suggestion
        '@next/next/no-assign-module-variable': 'error',
```
Looks like this was an error previously https://github.com/vercel/next.js/blob/b188fab3360855c28fd9407bd07c4ee9f5de16a6/packages/eslint-plugin-next/lib/index.js#L47

https://eslint.org/docs/latest/user-guide/configuring/rules

Reply:
You are correct. My apologies. Fixed. https://github.com/vercel/next.js/pull/38134/commits/cbf37a3016a6285016f8717d93bce83e37e1ebe6